webapp/channels/src/components/admin_console/server_logs/logs.tsx

```diff
@@ -137,23 +153,35 @@ export default class Logs extends React.PureComponent<Props, State> {
     };
 
     render() {
-        const content = this.props.isPlainLogs ? (
+        const ToggleView = (
+            <div className='logs-banner'>
+                <Toggle
+                    onText='Formatted'
+                    offText='Plain'
+                    toggled={this.state.isLocalFormattedView}
+                    onToggle={this.toggleMode}
+                />
+            </div>);
```

Comment:
The UX seems a bit confusing. When the formatted view is used, the toggle reads "Plain". Should that be inverted?
![Screenshot from 2024-04-22 12-14-27](https://github.com/mattermost/mattermost/assets/16541325/2beab792-9f1f-4511-b257-a8bb9d4f6c9a)